tests/test_models.py

```diff
@@ -66,6 +68,20 @@ def test_string_conversion(self, method):
         )
         self.assertEqual(method(customer), expected_to_str)
 
+    def test_site_association(self):
+        """
+        Test ``EnterpriseCustomer`` conversion to string.
+        """
+        faker = FakerFactory.create()
+        customer_uuid = faker.uuid4()
+        EnterpriseCustomerFactory(uuid=customer_uuid)
+
+        customer = EnterpriseCustomer.objects.get(uuid=customer_uuid)  # pylint: disable=no-member
+        site = Site.objects.get(domain="example.com")
+
+        self.assertEqual(customer.site, site)
+        self.assertListEqual([customer], list(site.enterprise_customers.all()))
```

Comment:
@e-kolpakov jumping here, did you mean to remove the test `test_site_association` here ?